app/models/bulkrax/oai_ia_entry.rb

```diff
@@ -24,6 +24,14 @@ def build_metadata
       parsed_metadata
     end
 
+    # OVERRIDE: v4.4.2 Bulkrax::ImportBehavior#add_collections
+    def add_collections
+      return if self.find_collection_ids.blank?
+
+      # we need this mapping to exist so that Bulkrax::ImportBehavior#build_for_importer calls #parent_jobs
+      self.parsed_metadata[related_parents_parsed_mapping] = self.find_collection_ids
```

Comment:
`related_parents_parsed_mapping` will not be set in "config/initializers/bulkrax.rb". there's a fallback of `'parents'` in the [#related_parents_parsed_mapping](https://github.com/samvera-labs/bulkrax/blob/v4.4.2/app/parsers/bulkrax/application_parser.rb#L89-L91) method though.